packages.basic.txt

```diff
@@ -7,46 +7,51 @@ fake-hwclock
 ntp
 avahi-daemon
 lm-sensors
+systemd udev sudo nano
+
+##Fix debconf: unable to initialize frontend: Dialog
+dialog
 
 # Language Packs (Fix utf8 & get rid of Ubuntu's annoying locale warings)
-language-pack-en
+#language-pack-en
```

Comment:
This and the other stuff you've commented out you need to justify.
Leave it in here even if it is already installed in the base ubuntu image.
Things might change in the future and then we'll have regressions that can be easily avoided by just leaving the package in here.